testsuite/gnat2goto/tests/range_check_non_bounded/test.adb

```diff
@@ -0,0 +1,8 @@
+procedure Test is
+   type Uint_8 is mod 2**8;
+   Uint_Var : Uint_8 := 5;
+   Int_Var : Integer;
+begin
+   Int_Var := Integer'(Integer (Uint_Var));
```

Comment:
Why both qualified expression _and_ typecast?

Reply:
Qualified expression alone doesn't even parse (expected `integer` type). And typecast alone leads hides the conversion (thus we wouldn't get the range-check).